library/cc/engine_builder.cc

```diff
@@ -190,8 +214,24 @@ EngineBuilder& EngineBuilder::enableH2ExtendKeepaliveTimeout(bool h2_extend_keep
   return *this;
 }
 
+EngineBuilder& EngineBuilder::setPorts(std::vector<uint32_t> ports) {
+  this->ports_ = ports;
+  return *this;
+}
+
+EngineBuilder& EngineBuilder::setIpvVersion(std::string ipv_version) {
```

Comment:
ditto. I'd think the only "new" think we're setting was the xDS hostname so I'm probably missing something